src/js/stores/MesosStore.js

```diff
@@ -0,0 +1,247 @@
+var EventEmitter = require("events").EventEmitter;
+var semver = require("semver");
+
+var AppDispatcher = require("../AppDispatcher");
+var Util = require("../helpers/Util");
+var InfoStore = require("../stores/InfoStore");
+var InfoActions = require("../actions/InfoActions");
+var InfoEvents = require("../events/InfoEvents");
+var MesosActions = require("../actions/MesosActions");
+var MesosEvents = require("../events/MesosEvents");
+
+const FILES_TTL = 60000;
+const STATE_TTL = 60000;
+const REQUEST_DELAY = 100;
+const MASTER_ID = "master";
+
+var version = null;
+var stateMap = {};
+var taskFileMap = {};
+var taskFileRequestQueue = [];
+var requestCount = 0;
```

Comment:
what kind of requests is this counter keeping track of? can we use a less generic name maybe?